daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -96,7 +91,6 @@ commandParser = subparser $ fold
     , command "deploy" (info (deployCmd <**> helper) deployCmdInfo)
     , command "ledger" (info (ledgerCmd <**> helper) ledgerCmdInfo)
     , command "run-jar" (info runJarCmd forwardOptions)
-    , command "run-platform-jar" (info runPlatformJarCmd forwardOptions)
```

Comment:
This command is used to run daml sandbox, navigator, json-api, etc so probably we need to keep it, just without special version logic.

Reply:
Good catch, I’ve instead switched the sdk config to call `run-jar` which is identical except for the version logic.